### ISSUE TITLE
g-menu-parent-indicator state changes not working all the time

### ISSUE THREAD

**Issue by kat05** [gantry/gantry5#72]
i have a home menu item with several child items (they are called: 'features', 'extensions', 'sample menu', etc) which are all parent items themselves.

when clicking on the > icon on any of the child items (for example on 'features'), the > icon turns into a x icon, which is fine.

as soon as i click the > icon on the next child item (for example 'extensions'), the x icon on the previously clicked child item ('features') should turn back into a > icon, but it does not.

basically i would expect only to see one of the child items having an x icon, but currently you can end up with several child items showing the x icon. maybe the pic explains it better.

![g5_menuparentindicator](https://cloud.githubusercontent.com/assets/5719486/7036210/1cdbbbd6-dd57-11e4-8cc8-101053633260.png)

**Comment by mahagr** [comment]
Related to #58
